website/docs/guides/container_service_extension.html.markdown

```diff
@@ -0,0 +1,494 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: Container Service Extension"
+sidebar_current: "docs-vcd-guides-cse"
+description: |-
+Provides guidance on configuring VCD to be able to install Container Service Extension.
```

Comment:
Just had an idea in preparation for the CSE.next work in an effort to separate these two projects/architectures:

Maybe we should state the major CSE version this guide targets everywhere in the headings, maybe even in examples/ folder name, this html filename, etc. So the scope for the users is super-clear. Then when CSE.next is out, we will have a separate new guide (with a different version in the filename, headings, etc.).

Something like:
container_service_extension**_3**.html.markdown
examplpes/container-service-extension**-3**
"Provides guidance on configuring VCD to be able to install Container Service Extension **3.x**."

What do you think?

CC: @Didainius

Reply:
Done. I have stated 3.1.x instead as it's the latest version (and the one this guide was tested against). I'm not very confident about 3.0.x.